spec/asar-spec.js

```diff
@@ -644,6 +699,13 @@ describe('asar package', function () {
           done()
         })
       })
+
+      it('can be promisified', (done) => {
+        util.promisify(ChildProcess.exec)('echo ' + echo + ' foo bar').then(({ stdout }) => {
```

Comment:
This is a promise so we don't need to use the `done` technique.  We can just return the promise in the test and the test will finish when the promise is resolved or rejected.

Reply:
@MarshallOfSound is there an example in the existing tests?